controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/JobControllerApiHandlerHelper.java

```diff
@@ -361,7 +361,15 @@ static HttpResponse overviewResponse(Controller controller, TenantAndApplication
                                                                      "/job/" + job.type().jobName()).normalize();
                 stepObject.setString("url", baseUriForJob.toString());
                 stepObject.setString("environment", job.type().environment().value());
-                if ( ! job.type().environment().isTest()) stepObject.setString("region", job.type().zone().value());
+                if ( ! job.type().environment().isTest()) {
+                    stepObject.setString("region", job.type().zone().value());
+                    var deployment = application.require(job.application().instance()).deployments().get(job.type().zone());
+                    var cloudAccount = deployment == null ? null : deployment.cloudAccount();
```

Comment:
Consider using the already present `status.deploymentFor(...)` to avoid reading the application data an additional time.

Reply:
Oh sorry, I read an `s` that wasn't there. This is fine.